### PR TITLE
LPS-121680 - Update @clayui dependencies

### PR DESCRIPTION
**Pull request by bryceosterhaus**
Updated @clayui/* dependencies for https://issues.liferay.com/browse/LPS-121680

### CODE REVIEW

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
Let's see if we can push this forward now that the release has been cut!

**Comment by jbalsas** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
ci:test:sf

**Comment by jbalsas** [comment]
(closed by mistake... butterfingers...)

**Comment by jbalsas** [comment]
I think this is as green as it gets...

File: modules/yarn.lock [hunk 1]
```diff
@@ -1093,7 +1093,16 @@
     "@clayui/layout" "^3.2.0"
     classnames "^2.2.6"
 
-"@clayui/autocomplete@3.1.12", "@clayui/autocomplete@^3.1.12":
+"@clayui/alert@3.5.0":
```
Comment (jbalsas) [inline modules/yarn.lock:1096]: Not deduped 😭

**Comment by jbalsas** [comment]
Okay, I'm putting this on hold... `yarn.lock` has more deletions than additions which is weird considering most `@clayui` packages are duplicated in there. There's plenty `d3` packages going away... any idea why's that? It might be unrelated, but would be good to double check.

@bryceosterhaus, could you rebase and  dedupe the `yarn.lock` file when you have the chance?

Thanks!

**Comment by bryceosterhaus** [comment]
@jbalsas looks like commerce packages didn't get properly updated, I might've ended `format-source-all` too soon... I'll get an update push today

**Comment by bryceosterhaus** [comment]
@jbalsas is commerce exempt from our formatter? I notice the packages in `commerce-frontend-js` do not get updated when running `ant format-source-all`, this is why the yarn.lock includes multiple versions of clay packages because its continuing to lock the version from commerce

**Comment by jbalsas** [comment]
> @jbalsas is commerce exempt from our formatter? I notice the packages in commerce-frontend-js do not get updated when running ant format-source-all, this is why the yarn.lock includes multiple versions of clay packages because its continuing to lock the version from commerce

Yeah, that's what I was going to tell you... chances are they have some exclusion preventing the rules from running. Look for `source-formatter-suppressions.xml` and `source-formatter.properties` files in the commerce modules to see what they're excluding.

We can do a manual update for now, but we should try to remove the JSON rules they're excluding and see if things work so we don't bump into this again (this is the second time now, I think 😢 )

**Comment by bryceosterhaus** [comment]
ci:test:relevant

**Comment by bryceosterhaus** [comment]
ci:test:sf

**Comment by bryceosterhaus** [comment]
@jbalsas I went ahead and disabled the commerce rules, ran `ant format-source-all`, and then discarded my changes to their SF rules. There was other complaints regarding certain scripts in the package.json files, so I just disabled them to update the clay dependencies. We will see if the tests complain....

**Comment by jbalsas** [comment]
Looks good enough now...

@FabioDiegoMastrorilli, @gianmarcobrunialti, @garofalog, could you please take a look at those issues with the `package.json` files?

We automatically update all clay dependencies through SF, so every time we pull in new clay updates we're hitting this issue where commerce is not properly updated because of the exclusions.

We can help figure out the issues if you need assistance!

Thanks!

**Comment by jbalsas** [comment]
ci:forward

**Comment by jbalsas** [comment]
@wincent, @bryceosterhaus, one thought I had on this is that we could move that task out of SF and into our custom rules. That way we can either run it from `yarn format` in the `modules` root or by calling a more specific script if we want like `liferay-npm-scripts update-foo`

What do you think?

**Comment by wincent** [comment]
I generally like the idea of moving stuff that affects us out of SF into something that we control, whenever its tractable to do so.

**Comment by bryceosterhaus** [comment]
> I generally like the idea of moving stuff that affects us out of SF into something that we control, whenever its tractable to do so.

👍  agreed

**Comment by wincent** [comment]
I have this in my little todo list app and was probably thinking of this when I wrote it...

Screenshot 2020-10-15 -091957-ttU4cocN@2x

**Pull request closed without merging**